src/app/public/modules/list-view-checklist/list-view-checklist.component.spec.ts

```diff
@@ -35,13 +34,48 @@ import { ChecklistState, ChecklistStateDispatcher, ChecklistStateModel } from '.
 import {
   SkyListModule,
   SkyListComponent,
-  SkyListToolbarModule
+  SkyListToolbarModule,
+  SkyListPagingModule
 } from '@skyux/list-builder';
 
 import { Observable } from 'rxjs/Observable';
 import { BehaviorSubject } from 'rxjs/BehaviorSubject';
```

Comment:
I know you didn't do these but could we clean up the import stylings while we are touching this file?

Reply:
done.